client/cypress/end-to-end/ballot-comparison-audit-board.spec.js

```diff
@@ -0,0 +1,121 @@
+before(() => cy.exec('./cypress/seed-test-db.sh'))
+
+describe('Audit Boards', () => {
+  const uuid = () => Cypress._.random(0, 1e6)
+  const id = 0
+
+  before(() => {
+    id = uuid()
+    cy.visit('/')
+    cy.loginAuditAdmin('audit-admin-cypress@example.com')
+    cy.get('input[name=auditName]').type(`TestAudit${id}`)
+    cy.get('input[value="BALLOT_COMPARISON"]').check({ force: true })
+    cy.findByText("Create Audit").click()
+    cy.wait(1000)
+    cy.contains("Audit Setup")
+    cy.fixture('CSVs/jurisdiction/sample_jurisdiction_filesheet.csv').then(fileContent => {
+      cy.get('input[type="file"]').first().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'sample_jurisdiction_filesheet.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').spread((firstButton, secondButton) => {
+      firstButton.click()
+    })
+    cy.contains("Upload successfully completed")   
+
+    cy.fixture('CSVs/contest/ballot_comparison_contests.csv').then(fileContent => {
+      cy.get('input[type="file"]').last().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'ballot_comparison_contests.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').click()
+    cy.findAllByText(/Upload successfully completed/).should('have.length', 2)
+    cy.wait(2000)
+    cy.findByText('Next').click()
+    cy.findAllByText('Target Contests').should('have.length', 2)
+    cy.get('input[type="checkbox"]').first().check({ force: true })
+    cy.findByText('Save & Next').click()
+    cy.findAllByText('Opportunistic Contests').should('have.length', 2)
+    cy.findByText('Save & Next').click()
+    cy.get('#state').select('AL')
```

Comment:
When possible, I think it would be better to use the testing library style of queries so that we are testing by what the user sees rather than our internal dom structure or css. For this one, I would use `cy.findByRole('select', {name: 'State'})` or if that didn't work `cy.findByLabelText('State')`.

When I set up Cypress, I installed a testing library plugin so that all the same queries we use in our unit tests are available.